### PR TITLE
IMU Feature

### PR DESCRIPTION
**Pull request by gr812b**
Added a 3D model mapper, which will allow playback of IMU data

Things to add:
- Current timestamp being played back
- Pause / Play
- Smarter way to play each point

You can test it at `localhost:5173/IMU`

### CODE REVIEW

File: front-end/src/components/model/ModelViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    replayControllerRef.current.on(handleEvent);
+
+    return () => {
+      replayControllerRef.current?.off(handleEvent);
+    };
+  }, [data, objectLoaded]);
+
+  const handleBinChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    if (!event.target.value) return;
+    fetchData(event.target.value).then(setData);
```
[the remark this answers is not in the document]
Comment (gr812b) [reply front-end/src/components/model/ModelViewer.tsx:55]: No

File: front-end/src/components/model/Eevee.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    if (boxHelperRef.current) {
+      boxHelperRef.current.update();
+    }
+  });
+
+  // Load the objects into the scene
+  useEffect(() => {
+    const loader = new OBJLoader();
+
+    loader.load('/Eevee.obj', (object) => {
```
Comment (camdnnn) [inline front-end/src/components/model/Eevee.tsx:28]: instead of using a hardcoded path could this be updated to use asset pathing

**Review by camdnnn — CHANGES_REQUESTED** [review]
Seems pretty good to me just one small pathing issue that should be updated

**Review by camdnnn — APPROVED** [review]
LGTM!

**Pull request merged**